tensorflow/lite/micro/tools/make/helper_functions.inc

```diff
@@ -395,6 +395,9 @@ $(PRJDIR)$(2)/esp-idf/%: tensorflow/lite/micro/tools/make/templates/esp/%.tpl
   $(eval MAIN_SRCS_RELATIVE := $(patsubst tensorflow/lite/micro/examples/$(2)/%,%,$(MAIN_SRCS)))
   $(eval TFLM_SRCS := $(filter-out tensorflow/lite/micro/examples/%,$(5)) $(3))
 
+# also remove person_detect_model_data.cc file
+  $(eval TFLM_SRCS := $(filter-out person_detect_model_data, $(TFLM_SRCS)))
```

Comment:
I'm guessing that you are adding this for an issue similar to https://github.com/tensorflow/tensorflow/issues/46860

My hope is that https://github.com/tensorflow/tensorflow/pull/47018 will have fixed your issue as well, and this exclusion will not be needed.